src/wp-includes/abilities/wp-core-abilities.php

```diff
@@ -0,0 +1,271 @@
+<?php
+/**
+ * Core Abilities registration.
+ *
+ * @package WordPress
+ * @subpackage Abilities_API
+ * @since 6.9.0
+ */
+
+declare( strict_types = 1 );
+/**
+ * Registers the core abilities categories.
```

Comment:
Nit: there was a larger discussion around the naming and the plural form should look like:

```suggestion
 * Registers the core ability categories.
```